Shaders/DisplayDepth.fx

```diff
@@ -52,36 +42,30 @@ uniform int iUIUpsideDown <
 	ui_category = "Preprocessor";
 	ui_type = "combo";
 	ui_label = "Upside Down";
```

Comment:
I think combo items have mean same as label.
suggest ui_label ="";

Reply:
Yes, having both is quite redundant. Will change that.